src/Func.cpp

```diff
@@ -2150,16 +2181,11 @@ Func &Func::gpu_tile(VarOrRVar x, VarOrRVar y, VarOrRVar z,
 Func &Func::shader(Var x, Var y, Var c, DeviceAPI device_api) {
     invalidate_cache();
 
-    reorder(c, x, y);
-    // GLSL outputs must be stored interleaved
-    reorder_storage(c, x, y);
```

Comment:
Oh, I see the call was removed down here. I'm not convinced this is the right thing to do.